shared/src/fixtures/arrow.rs

```diff
@@ -205,6 +261,38 @@ pub fn auto_create_table(server: &str, table: &str) -> String {
     format!("CREATE FOREIGN TABLE {table} () SERVER {server}")
 }
 
+pub fn create_foreign_table(server: &str, table: &str, fields: &[(&str, &str)]) -> String {
+    let fields_definition = fields
+        .iter()
+        .map(|(field_name, field_type)| format!("{field_name} {field_type}"))
+        .collect::<Vec<_>>()
+        .join(",");
+
+    format!("CREATE FOREIGN TABLE {table} ({fields_definition}) SERVER {server}")
+}
+
+pub fn setup_fdw_local_parquet_file_listing(
```

Comment:
This looks very similar to a helper function that we already have.

```rust
pub fn primitive_setup_fdw_local_file_listing(local_file_path: &str, table: &str) -> String {
    let create_foreign_data_wrapper = primitive_create_foreign_data_wrapper(
        "parquet_wrapper",
        "parquet_fdw_handler",
        "parquet_fdw_validator",
    );
    let create_server = primitive_create_server("parquet_server", "parquet_wrapper");
    let create_table = primitive_create_table("parquet_server", table);

    format!(
        r#"
        {create_foreign_data_wrapper};
        {create_server};
        {create_table} OPTIONS (files '{local_file_path}'); 
    "#
    )
}
```

It's not immediately obvious when one should be called vs. the other in our tests. If your function is an abstraction of `CREATE FOREIGN TABLE`, would you mind changing the other helper functions to use it as well?

Reply:
So currently the created table schema is hard coded with `primitive_setup_fdw_local_file_listing` 
This one allows for the specification of the table schema as the third parameter. 
Attempting to reuse it will propagate. I might have to change `primitive_create_table` -> `primitive_table_columns` and just returns the default columns.